### PR TITLE
fix authentication thymeleaf

### PR DESCRIPTION
**Pull request by haidt-0041**

### CODE REVIEW

**Review by hungnm-0039 — APPROVED** [review]

**Review by lampt-0038 — APPROVED** [review]

**Comment by haidt-0041** [comment]
ready

File: src/main/webapp/views/front/home.html [hunk 1]
```diff
@@ -12,8 +12,12 @@ <h1>Hello</h1>
 		<input type="submit" value="Sign in" />
 	</form>
 	<br>
-	<form action="#" th:action="@{j_spring_security_logout}" method="post">
+	<!-- <form action="#" th:action="@{j_spring_security_logout}" method="post">
```
Comment (hieudh12) [inline src/main/webapp/views/front/home.html:15]: xóa comment

File: src/main/webapp/views/front/home.html [hunk 2]
```diff
@@ -12,8 +12,12 @@ <h1>Hello</h1>
...
 		<input type="submit" value="Sign out" />
 	</form>
+	<br> -->
+	<form action="#" th:action="@{/register}" method="post">
+		<input type="submit" value="Register" />
+	</form>
 </div>
 </html>
```
Comment (hieudh12) [inline src/main/webapp/views/front/home.html:23]: Thiếu end of line

File: src/main/webapp/views/front/register.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+			</form>
+			<p>
+				<a href="/" th:href="@{/}">Back to home page</a>
+			</p>
+
+		</div>
+	</div>
+</div>
+</html>
```
Comment (hieudh12) [inline src/main/webapp/views/front/register.html:96]: End of line

File: src/main/webapp/views/front/register.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+<head>
+<meta charset="utf-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<title>Register</title>
+</head>
+
+<div layout:fragment="body">
+	<div class="login-box">
+		<div class="login-logo"></div>
+		<!-- /.login-logo -->
```
Comment (hieudh12) [inline src/main/webapp/views/front/register.html:14]: Xóa comment

File: src/main/webapp/views/front/login.html [hunk 5]
```diff
@@ -49,7 +49,11 @@
 				</div>
 			</form>
 			<p>
-				<a href="/" th:href="@{/}">Back to home page</a>
+				<a href="/" th:href="@{/}">Back to home page</a>
```
Comment (hieudh12) [inline src/main/webapp/views/front/login.html:52]: tab quá lớn
https://github.com/framgia/coding-standards/tree/master/eng/html_css

**Pull request merged**